.github/workflows/build_jni.yml

```
@@ -1,31 +1,31 @@
-          arch: ${{ matrix.arch }}
-      - name: Build JNI Library
-        run: |
-          cd cpp-src
-          cl /LD /I"$env:JAVA_HOME\include" /I"$env:JAVA_HOME\include\win32" ghost-common-jni.cpp /link /out:ghost-common-jni_vc2015_${{ matrix.arch == 'x64' && 'amd64' || 'aarch64'}}.dll
-      - name: Upload artifact
-        uses: actions/upload-artifact@v4
-        with:
-          name: jni-${{ matrix.arch == 'x64' && 'amd64' || 'aarch64'}}
-          path: cpp-src/*.dll
\ No newline at end of file
+#name: Build C++ JNI Library
```

actionlint (1.7.7)

workflow is empty

(syntax-check)